react/features/base/participants/types.ts

```diff
@@ -54,6 +52,10 @@ export interface ILocalParticipant extends IParticipant {
     userSelectedMicDeviceLabel?: string;
 }
 
+export interface IParticipantFeatures {
```

Comment:
Can you document the missing options here? I believe we have more of them.

Reply:
Will do this one trough another PR